src/__tests__/canonicalFields.int.test.ts

```diff
@@ -1,14 +1,55 @@
 import request from 'supertest';
 import { app } from '../app';
+import { db } from '../database';
+import type { Result } from 'tinypg';
 
 const agent = request.agent(app);
 
 describe('/canonicalFields', () => {
   describe('/', () => {
-    it('should return HTTP Status Code 200 OK', async () => {
+    it('returns an empty array when no data is present', async () => {
       await agent
         .get('/canonicalFields')
-        .expect(200);
+        .expect(200, []);
+    });
+
+    it('returns all canonical fields present in the database', async () => {
+      await db.sql('fixtures.canonicalFields.returnsAllCanonicalFields.seed');
+      await agent
+        .get('/canonicalFields')
+        .expect(
+          200,
+          [
+            {
+              createdAt: '2022-07-20T12:00:00.000Z',
+              dataType: 'string',
+              id: 1,
+              label: 'First Name',
+              shortCode: 'firstName',
+            },
+            {
+              createdAt: '2022-07-20T12:00:00.000Z',
+              dataType: 'string',
+              id: 2,
+              label: 'Last Name',
+              shortCode: 'lastName',
+            },
+          ],
+        );
+    });
```

Comment:
I like this more than the snap thing. It is so clear and simple! There is still the one level of indirection above to find the SQL (as opposed to seeing the test setup SQL directly in the test) but I think that is fine because it is consistent with the way we use the SQL files in the production code, etc., plus I don't think there are multi-line string literals in JavaScript yet.

Reply:
You called out the one thing that was still bugging me about this...

Side note: you can have multiline string literals in JS using backticks:

```
export const test = `
asdf
asdf
asdf
`;
```

I also don't like at all that we're muddying up our `queries` directory with test sql.  I would somewhat expect there to be fixtures for this kind of thing (especially if we consider the idea that a test's migrations might get long and complex), but... I'm gonna see what inline SQL looks like for this just for kicks.

Reply:
Moved it to inline SQL; it's the way to go.  Also if in future we ever feel the need to move a particularly long / complex DB setup to a `fixture` file, that fixture should be written alongside the test file rather than halfway across the world in the production `database/queries` directory.

Reply:
Very, very nice!!